boards/makerbase/mks_canable_v20/doc/index.rst

```diff
@@ -0,0 +1,171 @@
+.. zephyr:board:: mks_canable_v20
+
+Overview
+********
+
+The Makerbase MKS CANable V2.0 board features an ARM Cortex-M4 based STM32G431C8 MCU
+with a CAN, USB and debugger connections.
+Here are some highlights of the MKS CANable V2.0 board:
+
+- STM32 microcontroller in LQFP48 package
+- USB Type-C connector (J1)
+- CAN-Bus connector (J2)
+- ST-LINK/V3E debugger/programmer header (J4)
+- USB VBUS power supply (5 V)
+- Three LEDs: red/power_led (D1), blue/stat_led (D2), green/word_led (D3)
+- One push-button for RESET
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell.
+
+The LED red/power_led (D1) is connected directly to on-board 3.3 V and not controllable by the MCU.
+
+More information about the board can be found at the `MKS CANable V2.0 website`_.
+It is very advisable to take a look in on user manual `MKS CANable V2.0 User Manual`_ and
+schematic `MKS CANable V2.0 schematic`_ before start.
+
+More information about STM32G431KB can be found here:
+
+- `STM32G431C8 on www.st.com`_
+- `STM32G4 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr ``mks_canable_v20`` board target supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | universal-serial-bus                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| FDCAN     | on-chip    | can                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/makerbase/mks_canable_v20/mks_canable_v20_defconfig`
+
+
+Connections and IOs
+===================
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- CAN_RX/BOOT0 : PB8
+- CAN_TX : PB9
+- D2 : PA15
+- D3 : PA0
+- USB_DN : PA11
+- USB_DP : PA12
+- SWDIO : PA13
+- SWCLK : PA14
+- NRST : PG10
+
+For more details please refer to `MKS CANable V2.0 schematic`_.
+
+System Clock
+------------
+
+The MKS CANable V2.0 system clock is driven by internal high speed oscillator.
+By default system clock is driven by PLL clock at 170 MHz,
+the PLL is driven by the 16 MHz high speed internal oscillator.
+
+The FDCAN1 peripheral is driven by PLLQ, which has 170 MHz frequency sam as main
+system clock.
```

Comment:
> The CAN FD peripheral should not run at 170 MHz. Please aim for a CAN core frequency of 80 MHz for interoperability reasons (recommended by CiA).
> 

This is a good point. I see it in CiA 601-2 CAN  and here: [CiA 601-2 CAN ](https://www.can-cia.org/fileadmin/cia/documents/publications/cnlm/march_2018/18-1_p28_recommendation_for_the_canfd_bit-timing_holger_zeltwanger_cia.pdf)

Unfortunatel I can not find other way than setting main clock to 160MHz s.t. CAN clock will be 80MHz. But since the board is specifically aimed for CAN it would be totally fine.

> Also, what does "sam" mean in the sentence above?

Shall be "same". Corrected.